tests/test_fixtures/rule_based_profiler/example_notebooks/BasicExample_RBP_Instantiation_and_running.ipynb

```diff
@@ -0,0 +1,1778 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### `ValueSetMultiBatchParameterBuilder`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `ValueSetMultiBatchParameterBuilder` is able to build a value set across multiple Batches (or just one Batch). "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "domain: Domain = Domain(domain_type=MetricDomainTypes.COLUMN, domain_kwargs = {'column': 'vendor_id'})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 38,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# instantiating a new parameter container, since it can contain the results of more than one ParmeterBuilder. \n",
+    "parameter_container: ParameterContainer = ParameterContainer(parameter_nodes=None)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 39,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "value_set_parameter_builder: ValueSetMultiBatchParameterBuilder = (\n",
+    "    ValueSetMultiBatchParameterBuilder(\n",
+    "        data_context=data_context,\n",
+    "        batch_request=single_batch_batch_request,\n",
+    "        metric_domain_kwargs=domain.domain_kwargs,\n",
+    "        name=\"my_value_set\",\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 40,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "3b87eff676ce4a59a8ff7b7af24415f6",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Calculating Metrics:   0%|          | 0/4 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "value_set_parameter_builder.build_parameters(\n",
+    "    parameter_container=parameter_container,\n",
+    "    domain=domain,\n",
+    "    parameters = {domain.id: parameter_container}\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 41,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'parameter': {'parameter': {'my_value_set': {'value': [1, 2, 4], 'details': {'metric_configuration': {'metric_name': 'column.distinct_values', 'domain_kwargs': {'column': 'vendor_id'}, 'metric_value_kwargs': None, 'metric_dependencies': None}, 'num_batches': 1}}}}}\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(parameter_container.parameter_nodes)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`my_value_set[value]` now contains a list of 3 values, which is a list of all `vendor_ids` across 12 Batches in the 2018 `taxi_data` dataset."
```

Comment:
You also probably will need to change the `3 values` to the number of uniques from all 12 batches.
```suggestion
    "`my_value_set[value]` now contains a list of 3 values, which is a list of all unique `vendor_ids` across 12 Batches in the 2018 `taxi_data` dataset."
```